content/en/docs/tutorials/configuration/pod-sidecar-containers.md

```diff
@@ -0,0 +1,148 @@
+the native implementation of this feature in Kubernetes, however, is new. And as with every new feature,
+adopting this feature may present certain challenges.
+
+This article explore challenges and solution can be experienced by end users as well as
+by authors of sidecar containers.
+
+## Benefits of a built-in sidecar containers
+
+There are many benefits for the built-in sidecar containers.
+
+1. Built-in sidecar containers can run before Init containers.
+2. It is guaranteed that built-in sidecar containers finish AFTER all the regular containers completion.
+3. With Jobs, when Pod's `restartPolicy=OnFailure` or `restartPolicy=Never`,
+   built-in sidecar containers will not block Pod completion.
+4. Also, with Jobs, build-in sidecar containers will keep being restarted, even if regular containers would not with Pod's `restartPolicy=Never`.
+
+These benefits are drastic, and lead to faster adoption that we see with other Kubernetes features.
+
+## Adopting sidecar containers
+
+The `SidecarContainers` feature is in beta state starting Kubernetes version `1.29` and is enabled by default.
+Some clusters may have this feature disabled or have software installed that is incompatible with the feature.
+
+When this happens, the Pod may be rejected or the sidecar containers may block Pod startup, rendering the Pod useless.
+This condition is easy to detect as Pod simply get stuck on initialization. However, it is rarely clear what caused the problem.
+
+Here are the considerations and troubleshooting steps that one can take while adopting sidecar containers for their workload.
+
+**Ensure the feature gate is enabled.**
+
+As a very first step, make sure that both - API server and Nodes were upgraded to the version `1.29+`.
+The feature will break on clusters where Nodes are running earlier versions where it is not enabled.
+
+The feature gate enablement should also be validated for both - API server AND Nodes.
+
+One of the ways to check the feature gate enablement is running a command like this:
+
+- For API Server 
+  `kubectl get --raw /metrics | grep kubernetes_feature_enabled | grep SidecarContainers`
+- For the individual node: 
+  `kubectl get --raw /api/v1/nodes/<node-name>/proxy/metrics | grep kubernetes_feature_enabled | grep SidecarContainers`
+
+If you see something like: `kubernetes_feature_enabled{name="SidecarContainers",stage="BETA"} 1`,
+it means that the feature is enabled.
+
+**Check for 3rd party tooling and mutating webhooks**
+
+If you experience issues when validating the feature, it may be an indication that one of the
+3rd party tools or mutating webhooks are broken.
+
+The `SidecarContainers` feature adds a new field that is define in Kubernetes `1.28` API.
+Some tools or mutating webhooks might have been built with an earlier version of Kubernetes API.
+If tools pass the unknown fields as-is or mutating webhooks are using the recommended `Patch` method to mutate Pods, it will not be a problem.
```

Comment:
Maybe you meant:
```markdown
-If tools pass the unknown fields as-is or mutating webhooks are using the recommended `Patch` method to mutate Pods, it will not be a problem.
+If tools pass the unknown fields as-is or mutating webhooks are using the recommended `PATCH` method
+to mutate Pods, it will not be a problem.
```
HTTP verbs are typically all-uppercase. However, **mutating webhooks don't use PATCH** because they are an HTTP callout (and response) that modifies the request body during admission.

Reply:
I rephrased